ci/build-helpers.groovy

```diff
@@ -81,7 +81,7 @@ def build(name, dockerFile, targetPlatform, objDir, params, buildId) {
 
         image = stage('docker build base') {
             docker.build('ua-build-base', '-f build/Base.dockerfile ./build/ --build-arg user=`whoami` --build-arg UID=`id -u` --build-arg GID=`id -g`')
-            docker.build("ua-build-${name.toLowerCase()}", "-f build/${dockerFile} ./build")
+            docker.build("ua-build-${name.toLowerCase()}", "-f build/${dockerFile} ./build --build-arg IPFS_GATEWAY=http://kria.cliqz:8080")
```

Comment:
does this mean kria has to run ips gateway?

Reply:
Yes, at the moment it will. We can also remove this and use a public IPFS gateway (default is cloudflare-ipfs), but then we lose the benefit of local caching.

Reply:
Gotcha. Can you document somewhere how it should be set up? 

Should we have ansible script for kria maybe? 

cc @philipp-classen